client/webserver/locales/en-us.go

```diff
@@ -27,7 +27,7 @@ var EnUS = map[string]string{
 	"Submit":                         "Submit",
 	"Confirm Registration":           "Confirm Registration",
 	"app_pw_reg":                     "Enter your app password to confirm DEX registration.",
-	"reg_confirm_submit":             `When you submit this form, <span id="feeDisplay"></span> DCR will be spent from your Decred wallet to pay registration fees.`, // TODO for multi-asset reg
+	"reg_confirm_submit":             `When you submit this form, it will spent from your chosen wallet to pay the registration fee.`,
```

Comment:
```suggestion
	"reg_confirm_submit":             `When you submit this form, it will spend from your chosen wallet to pay the registration fee.`,
	"accepted_fee_assets":            "This DEX accepts fee payment with the following assets. Select one:",
```